autotest.pm

```diff
@@ -313,9 +313,9 @@ sub start_process () {
             $0 = "$0: autotest";
             my $line = <$isotovideo>;
             if (!$line) {
-                _exit(0);
+                _exit(0);    # uncoverable statement
             }
-            print "GOT $line\n";
+            bmwqemu::diag("GOT $line\n");
```

Comment:
I agree with the change but it shouldn't be hidden in the commit "Add subtest for autotest.pm". Please make that a separate commit

Reply:
Pushed this change in a new commit